packages/calypso-build/bin/calypso-build.js

```diff
@@ -11,6 +11,8 @@
 const fs = require( 'fs' );
 const path = require( 'path' );
 
+process.env.BROWSERSLIST_ENV = process.env.BROWSERSLIST_ENV || 'wpcom';
```

Comment:
Does it always check the wp-calypso root package.json to find the `wpcom` browserlist config? Why would that not be in the local package.json for the package?

Reply:
The docs says

> browserslist key in package.json file in current or parent directories.

I think the key here is "current directory", not "directory of the code being transpiled". In theory we could store that info in local `package.json`, assuming we always do `cd apps/editor-toolkit && yarn build` and not `yarn workspace @automattic/wpcom-editing-toolkit build` (I think, I'm not 100% sure).

Having it in the root `package.json` is the safest option, but I'd agree that ensuring each package has its own `browserslist` config may keep things more isolated.

Reply:
Ah, I meant that if `calypso-build` is published to `npm`, how does it know that "wpcom" translates to the browserlist definition which lives "outside" of the `calypso-build` package?

Reply:
Ah, I meant that if `calypso-build` is published to `npm`, how does it know that "wpcom" translates to the browserlist definition which lives "outside" of the `calypso-build` package?

Reply:
I'm having second thoughts about this change: It doesn't make sense to assume that every consumer of `@automattic/calypso-build` (a public package) is building for WPCom, the default may not make sense for them. So maybe it is better to not have any default.

In our specific case it would mean that either package using `calypso-build` has a valid `browserslist` config, or if they use environments they are responsible for setting the right `BROWSERSLIST_ENV`

Reply:
Yeah, I think that makes sense